docker/conf/docker-compose.yml.dist

```diff
@@ -94,6 +94,14 @@ services:
         environment:
             - discovery.type=single-node
 
+    kibana:
+        image: docker.elastic.co/kibana/kibana-oss:7.6.1
```

Comment:
Hmm, It's impossible for me to bring up this container. it ends with error
`ERROR: manifest for docker.elastic.co/kibana/kibana-oss:7.6.1 not found: manifest unknown: manifest unknown`

Also, I don't see this tag in https://container-library.elastic.co/r/kibana/kibana-oss

Reply:
That is right, there is only `7.6.0`. Now it is fixed.